examples/aws/rds-s3/README.md

```diff
@@ -0,0 +1,148 @@
+# Kustomize Manifests for RDS and S3
+
+## Overview
+This Kustomize Manifest can be used to deploy Kubeflow Pipelines and Katib with RDS and S3.
+
+Follow the [install](#install) steps below to configure and deploy the Kustomize manifest.
+
+## Provisioning AWS Resources 
+
+### Create S3 Bucket
+
+Run this command to create S3 bucket by changing `<YOUR_S3_BUCKET_NAME>` and `<YOUR_CLUSTER_REGION` to the preferred settings.
+
+```
+export S3_BUCKET=<YOUR_S3_BUCKET_NAME>
+export CLUSTER_REGION=<YOUR_CLUSTER_REGION>
+aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
+```
+
+### Create RDS Instance
+
+Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance.
+
+## Install
```

Comment:
One liner description of the section to follow? are we going to have single and individual component installation?

add a step to clone this repo and checkout this branch

Reply:
removing single component installation since users familiar with kustomize will know how to apply single component, while users unfamiliar would get confused with the additional options